libs/payments/ui/src/lib/constants.ts

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+export enum BannerVariant {
```

Comment:
If this is specific to the banner component, it should be colocated with the component.

Reply:
`BannerVariant` was placed in this file as there was an issue with Nextjs finding the module.

Ref: https://github.com/vercel/next.js/issues/58776